src/axom/sidre/core/Group.cpp

```diff
@@ -1991,22 +2102,26 @@ Group* Group::createGroupAndLoad(std::string& group_name,
  *
  *************************************************************************
  */
-void Group::loadExternalData(const std::string& path)
+bool Group::loadExternalData(const std::string& path)
 {
   Node n;
   createExternalLayout(n);
+  ConduitErrorSuppressor checkConduitCall(getDataStore());
 
 #ifdef AXOM_USE_HDF5
   // CYRUS'-NOTE, not sure ":" will work with multiple trees per
   // output file
-  conduit::relay::io::hdf5_read(path + ":sidre/external", n);
+  checkConduitCall(
+    [&] { conduit::relay::io::hdf5_read(path + ":sidre/external", n); });
 #else
   AXOM_UNUSED_VAR(path);
   SLIC_WARNING(SIDRE_GROUP_LOG_PREPEND
                << "External data not loaded. "
                << "This function requires hdf5 support. "
                << " Please reconfigure with hdf5.");
 #endif
+
+  return !(getDataStore()->getConduitErrorOccurred());
```

Comment:
This should also return false if the `#else` code section is used--if no HDF5, no data is loaded.

Reply:
thanks---making this change.